lib/install/template.rb

```diff
@@ -11,10 +11,12 @@
 directory "#{__dir__}/config/loaders/core", "config/webpack/loaders"
 copy_file "#{__dir__}/config/.postcssrc.yml", ".postcssrc.yml"
 
-append_to_file ".gitignore", <<-EOS
+if File.exists?(".gitignore")
+  append_to_file ".gitignore", <<-EOS
```

Comment:
It is nicely aligned but the content of that string is now different that what it was